etcdserver/api/v3rpc/keepalive/keepalive.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017 The etcd Authors
```

Comment:
instead of a separate package under etcdserver, could this all go in `embed` and extend `v3rpc.Server()` to accept a list of server options?